core/src/test/java/com/sequenceiq/cloudbreak/converter/v4/blueprint/BlueprintV4RequestToBlueprintConverterTest.java

```diff
@@ -3,6 +3,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
```

Comment:
Could you please migrate this class to JUnit 5?

Reply:
done